src/main/java/com/epam/lab/paymentsystem/controller/GlobalControllerAdvice.java

```diff
@@ -0,0 +1,123 @@
+package com.epam.lab.paymentsystem.controller;
+
+import com.epam.lab.paymentsystem.dto.AccountDto;
+import com.epam.lab.paymentsystem.dto.CardDto;
+import com.epam.lab.paymentsystem.dto.OperationDto;
+import com.epam.lab.paymentsystem.dto.UserDto;
+import com.epam.lab.paymentsystem.exception.AccountArgumentException;
+import com.epam.lab.paymentsystem.exception.CardArgumentException;
+import com.epam.lab.paymentsystem.exception.LoginAlreadyExistsException;
+import com.epam.lab.paymentsystem.exception.MoneyTransferException;
+import com.epam.lab.paymentsystem.exception.ResourceNotFoundException;
+import com.epam.lab.paymentsystem.service.AccountService;
+import com.epam.lab.paymentsystem.service.CardService;
+import com.epam.lab.paymentsystem.service.UserService;
+import java.util.Map;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.servlet.ModelAndView;
+
+@ControllerAdvice(basePackages = {"com.epam.lab.paymentsystem.controller"})
+public class GlobalControllerAdvice {
+
+  private static final String MESSAGE_EXCEPTION = "messageException";
+  private static final String USER_LOGIN_PATH_VARIABLE = "userLogin";
+  private static final String ACCOUNT_ID_PATH_VARIABLE = "accountId";
+  private static final String CARD_ID_PATH_VARIABLE = "cardId";
+  @Autowired
+  private UserService userService;
+  @Autowired
+  private AccountService accountService;
+  @Autowired
+  private CardService cardService;
+
+  /**
+   * Throws Http Response NOT_FOUND.
+   *
+   * @param pathVariables map of path variables
+   */
+  @ModelAttribute
+  public void advise(@PathVariable Map<String, String> pathVariables) {
```

Comment:
I feel like you could use some pattern for better code but I cannot provide you with right information. So this method considers to be approved if @nikandtos approves it